.gitlab-ci.yml

```diff
@@ -53,6 +72,27 @@ sniffers:sonarqube:
   - master
   allow_failure: true
 
+prepare:nodejs:
+  cache:
+    key: "front-packages"
+    paths:
+      - ${THEME_PATH}/node_modules/
+  stage: prepare
+  image: node:lts-alpine
+  before_script:
+  - date
+  - pwd
+  - node -v
+  - yarn -v
+  script:
+  - cd ${THEME_PATH}
+  - yarn install --no-optional --prod
+  - yarn run build --verbose
```

Comment:
Script part of this CI step (prepare:nodejs) could be updated to use [`make front`](https://github.com/skilld-labs/skilld-docker-container/blob/5525d103ffa9d9fae437cc94b0781b2213c5601f/scripts/makefile/front.mk#L23) since it's executing the same commands

Reply:
Follow up issue created in contrib board